Astronomer Enterprise v0.23 Release Notes

paolaperaza opened the pull request:
@astrorfox @vishwas-astro This is what I got for an initial draft of Astronomer v0.23.9 Enterprise release notes. This largely includes descriptions for new features available in Astronomer Cloud that had not yet been shipped to Enterprise (and corresponding documentation links/references).

A few questions:
- I primarily relied on GitHub labels (issues on 0.16+ that weren't already backported), but there is unfortunately no source of truth (that I know of) for 100% of changes included in this release. Is there a better place for me to look?
- I get the sense that I missed Houston-related changes that we never communicated to Cloud but perhaps do apply to Enterprise. I filtered for the `area/houston-api` label in GitHub and found some, but I can't be sure I got them all.
- Does it make sense to include a section for Astronomer CLI latest support?
- We unfortunately do not have an "Upgrade to Airflow 2.0" doc written for Enterprise - totally on me. Will prioritize getting that created this week and linked in release notes.

Let me know what you all think.

> Note: All internal docs links are to `/stable` path but will be updated to `/v0.23` once that new directory is published.

vishwas-astro commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,107 @@ description: "Astronomer Enterprise release notes."
+In contrast with Astronomer v0.16, Astronomer v0.23+ does not require that a release be shipped with explicit support for any Astronomer Certified version or set of Docker images.
 
-#### Bug Fixes and Improvements
+As soon as a version of Astronomer Certified is published to our [updates service](http://updates.astronomer.io/astronomer-certified), all corresponding Astronomer Certified images and tags will immediately become available for selection in the Astronomer UI and CLI.
```
It is not immediately available but will become available when the cron job runs that day/night in the customer environment. We should document the importance of that cron job somewhere.

paolaperaza replied:
Yea, I can add more detail here and potentially document further in "Manage Airflow Versions."

vishwas-astro commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,107 @@ description: "Astronomer Enterprise release notes."
+- A set of error messages to alert you if a deploy has failed or was otherwise _not_ complete.
 
-#### Airflow Chart: Improved Upgrade Process
+#### Houston API Improvements
```
I am trying to remember if we pushed out the KEDA mutation to an 0.16 patch release. If so, then we may not need to document it here again. But that was a houston improvement added.

paolaperaza replied:
@vishwas-astro Yea, looks like we backported it to v0.16.10. Enterprise v0.16 release notes have a line-item that reads:

> - Improvement: New Houston API mutation that allows users to enable or disable KEDA (`updateDeploymentKedaConfig`)

We're working on docs and evidently haven't made this a first-class citizen feature, but do you think we need to discuss it at all in v0.23? Have we made any KEDA improvements since then?

astrorfox replied:
I'd leave it out for now and then we can announce as an experimental feature when the doc is ready

paolaperaza replied:
Cool, that sounds good @astrorfox.

vishwas-astro commented:
What about the simplified kubernetes executor, do we need to mention it here somewhere?

Also, may be look through the airflow chart repo. We upgraded our dependency on it from 0.15.x(7 i believe) to 0.18.y. so, may want to include some of the improvements/fixes from there.

Do we need to talk about the NATS stuff being introduced as a dependency or an improvement now?

vishwas-astro commented:
What about mentions of the upgrade process?

astrorfox commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,107 @@ description: "Astronomer Enterprise release notes."
+#### Platform Support for Airflow 2.0
 
-For step-by-step guidelines, refer to ["Forward Logs to S3"](https://www.astronomer.io/docs/enterprise/stable/monitor/logs-to-s3/).
+Astronomer v0.23.9 offers full support for [Airflow 2.0](https://www.astronomer.io/blog/introducing-airflow-2-0/) on Astronomer. Already available for local development, Airflow 2.0 is a momentous open-source release that includes a refactored Scheduler, over 30 UI/UX improvements, a new REST API and much more.
```
Given that this is the first release of 0.23.x, should we refer to these broad statements as 0.23 (no patch number)?

paolaperaza replied:
@astrorfox Yea, I can keep it to `0.23` for general statements here. Done!

astrorfox commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,107 @@ description: "Astronomer Enterprise release notes."
+#### Support for Multiple Schedulers (_Airflow 2.0+_)
 
-### v0.16.6
+As mentioned above, Airflow 2.0 allows users to provision multiple Airflow Schedulers for ultimate high-availability and scale. In tandem with full support for Airflow 2.0 on Astronomer, v0.23 supports the ability to provision up to 4 Schedulers via the Astronomer UI for Airflow Deployments running Airflow 2.0+.
```
```suggestion
Airflow 2.0 allows users to provision multiple Airflow Schedulers for ultimate high-availability and scale. In tandem with full support for Airflow 2.0 on Astronomer, v0.23 supports the ability to provision up to 4 Schedulers via the Astronomer UI for Airflow Deployments running Airflow 2.0+.
```

astrorfox commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,107 @@ description: "Astronomer Enterprise release notes."
+Astronomer v0.23 formally introduces **Airflow Version** to the Astronomer UI, CLI and API for an enhanced version selection and Airflow upgrade experience.
 
-For example, you might want to add an Airflow alert to fire if a Deployment has been using over 50% of its pod quota for over 10 minutes or a Platform alert if more than 2 Airflow Schedulers don't heartbeat for more than 5 minutes. Here's what your `config.yaml` might look like:
+To see version of Airflow your Deployment is running, refer to the corresponding **Settings** page of the Astronomer UI. If you're interested in upgrading versions, you can initialize the upgrade process via the Astronomer UI or CLI for a guided experience. From there, you'll receive instructions to update the Astronomer Certified (AC) image in your Dockerfile and complete the process.
```
```suggestion
To see the version of Airflow your Deployment is running, refer to the corresponding **Settings** page of the Astronomer UI. If you're interested in upgrading versions, you now initialize the upgrade process via the Astronomer UI or CLI for a guided experience. From there, you'll receive instructions to update the Astronomer Certified (AC) image in your Dockerfile and complete the process.
```

astrorfox commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,107 @@ description: "Astronomer Enterprise release notes."
+#### Real-time Support for Latest Astronomer Certified Versions
 
-> **Note:** These alerts are NOT Airflow task-level alerts that require definition at the Airflow code level. For more information on how to set up `email_on_failure` within Airflow, for example, refer to our ["Airflow Alerts" doc](/docs/enterprise/stable/customize-airflow/airflow-alerts/).
+In contrast with Astronomer v0.16, Astronomer v0.23+ does not require that a release be shipped with explicit support for any Astronomer Certified version or set of Docker images.
```
```suggestion
In contrast with v0.16, Astronomer Enterprise v0.23+ does not require that a release be shipped with explicit support for any Astronomer Certified version or set of Docker images. Decoupling these components helps you stay current with less toil.
```

astrorfox commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,107 @@ description: "Astronomer Enterprise release notes."
+- `createDeployment`
 
-More specifically, this changes the behavior of our API's `updateDeployment` mutation to perform the Airflow Helm Chart version upgrade only if/when a Houston config is updated. [Source Code here](https://github.com/astronomer/docs/blob/9cd8da9a4382bc89d7b8c07f5585ad1daa64a250/enterprise/next/reference/update-deployment-index.js#L86).
+If you're calling either of those mutations in your current workflow, Astronomer v0.23 will introduce a breaking change.
```
Is this true? Seems to break semver if so.

vishwas-astro replied:
Yes @astrorfox that is true. We have really not been a very API first product so far and have made innumerable breaking changes in Houston, I would say, between 0.1 (or wherever we started) to 0.23 (where we are now).

So far, we have typically combined the concepts of major and minor of semver into "minor" and have for some reason resisted moving to an Astronomer 1.0 offering.

Once we move to 1.0, we can start strictly enforcing the rules of Semantic Versioning.

astrorfox replied:
Can we not version the API? In other words, introduce v2 of the API in 0.x?

astrorfox commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,107 @@ description: "Astronomer Enterprise release notes."
+If you're calling either of those mutations in your current workflow, Astronomer v0.23 will introduce a breaking change.
 
-#### Bug Fixes and Improvements
+As we strive to polish and standardize the schema more generally, our API will undergo rapid and potentially breaking changes over the next few releases. The Astronomer CLI and UI will continue to stay in sync with API changes and is our recommended way to interact with the platform.
```
Why would we not increment the API version to prevent this?

vishwas-astro replied:
What good would incrementing the major of the API version do when all the customers are doing is an upgrade to an Astronomer revision, lets say from 0.16 to 0.23, and the underlying API that gets shipped with the software starts breaking?

astrorfox replied:
Incrementing the API would have a v1 and v2 API running in parallel

astrorfox replied:
Will take up this conversation separately offline

paolaperaza commented:
> What about the simplified kubernetes executor, do we need to mention it here somewhere?
> 
> Also, may be look through the airflow chart repo. We upgraded our dependency on it from 0.15.x(7 i believe) to 0.18.y. so, may want to include some of the improvements/fixes from there.
> 
> Do we need to talk about the NATS stuff being introduced as a dependency or an improvement now?

@vishwas-astro The simplified Kubernetes Executor came with Airflow 2.0 - or do you mean something we (Astronomer) added on top of that? I thought that the main thing we added was support for the new Executor in our Airflow chart, but that's more behind-the-scenes enablement for the Airflow feature. Would you add anything on top of that?

vishwas-astro commented:
> > What about the simplified kubernetes executor, do we need to mention it here somewhere?
> > Also, may be look through the airflow chart repo. We upgraded our dependency on it from 0.15.x(7 i believe) to 0.18.y. so, may want to include some of the improvements/fixes from there.
> > Do we need to talk about the NATS stuff being introduced as a dependency or an improvement now?
> 
> @vishwas-astro -
> 
> 1. The simplified Kubernetes Executor came with Airflow 2.0 - or do you mean something we (Astronomer) added on top of that? I thought that the main thing we added was support for the new Executor in our Airflow chart, but that's more behind-the-scenes enablement for the Airflow feature. Would you add anything on top of that?

I think you are right. All of it should be behind the scenes. I was just thinking if there are any nuances to K8sExecutor deployments that users can control/modify, we should capture them. May be check once with Daniel Imbermann.

> 2. Good call - I'll look through Airflow chart changes, but to some extent those seem like they enable other features? Do I need to call anything out in particular?

I think this airflow chart improvement could be useful to mention: https://github.com/astronomer/issues/issues/1410

> 3. I covered NATs stuff in the `A New "Deploment Status" Framework` section, although it doesn't go into much technical detail. Do you think I should add more there?
Yah, that is good enough.

vishwas-astro commented:
@paolaperaza @astrorfox Do we need to indicate any Open Shift support in these release notes?

paolaperaza commented:
@astrorfox @vishwas-astro Updates:

- Added Celery Worker Update Strategy with description (to Cloud + Enterprise) - checked with Jim on this ✅ 
- Added a line-item for "Improved OpenShift Support" ✅ 
- Added a line-item for "Support for Azure Database for PostgreSQL - Flexible Server"

I have not gotten clarity on Houston changes. @vishwas-astro Would you say to leave as is?

vishwas-astro commented:
> I have not gotten clarity on Houston changes. @vishwas-astro Would you say to leave as is?
Yes, that is fine.

vishwas-astro commented on cloud/stable/06_resources/01_release-notes.md:
```diff
@@ -6,6 +6,12 @@ description: "Astronomer Cloud Release Notes."
+
+Release Date: January 18, 2021
+
+This release was exclusively infrastructure maintenance in preparation for Astronomer Enterprise v0.23.9 release. Stay tuned for v0.24!
```
I dont think we should say stay tuned for 0.24 .. Dont know when we will ship that just yet.

jwitz replied:
```suggestion
This release was exclusively infrastructure maintenance for the release of Astronomer Enterprise v0.23. Stay tuned for v0.24!
```

jwitz commented on cloud/stable/06_resources/01_release-notes.md:
```diff
@@ -60,7 +66,7 @@ As of Astronomer v0.23.5, we:
 - Support Airflow's ["upgrade check"](https://airflow.apache.org/docs/apache-airflow/stable/upgrade-check.html) in the Astronomer CLI (`$ astro dev upgrade-check`)
 
-The migration path to Airflow 2.0 on Astronomer follows all recommendations published by the Apache Airflow project. We expect to publish an official "Upgrade to Airflow 2.0 on Astronomer" guide as soon as Airflow 2.0 is generally available.
+To get started, read [Upgrade to Apache Airflow 2.0 on Astronomer](https://www.astronomer.io/docs/cloud/stable/customize-airflow/upgrade-to-airflow-2). The migration path to Airflow 2.0 on Astronomer follows all recommendations published by the Apache Airflow project.
```
Not seeing a link to these "published" recommendations here or in the Upgrade doc, maybe we should cut that word lest it send someone on a wild goose chase.

paolaperaza replied:
Ah yea hm. Those recommendations are here: https://airflow.apache.org/docs/apache-airflow/stable/upgrading-to-2.html

I'll link to them!

jwitz commented on cloud/stable/06_resources/01_release-notes.md:
```diff
@@ -81,6 +87,16 @@ Astronomer v0.23 supports Airflow images that are pulled from _either_ Docker re
 
+#### Improved Celery Worker Update Strategy
+
+Astronomer v0.23 includes an improvement to the process by which new Celery Workers are created and begin to pick up tasks following a deploy to an Airflow Deployment on Astronomer.
```
```suggestion
Astronomer v0.23 includes an improvement to how Celery Workers are created when deploying to an Airflow Deployment on Astronomer.
```
We don't describe how the actual task intake has improved as much as we do the worker creation, so I would just focus on that for the introduction.

paolaperaza replied:
I'd say the task intake is critical! They workers can start taking up tasks as soon as the worker is created, but really the former is the primary value add so I do want to emphasize that.

jwitz commented on cloud/stable/06_resources/01_release-notes.md:
```diff
@@ -81,6 +87,16 @@ Astronomer v0.23 supports Airflow images that are pulled from _either_ Docker re
+
+Previously, the number of Celery Worker replicas that could be immediately created following a deploy was restricted to 25% of the total number of desired Celery Workers - until the original Workers shut down. This meant that it took longer for replacement Celery Workers to be created and start picking up Airflow tasks. Defined as `maxSurge`, that percentage is now set to 100%, allowing the maxim ...
+
+For users, this change will result in a quicker, more efficient and more graceful deploy process.
```
```suggestion
For users, this change will result in a quicker, more efficient, and more graceful deploy process.
```

jwitz commented on cloud/stable/06_resources/01_release-notes.md:
```diff
@@ -81,6 +87,16 @@ Astronomer v0.23 supports Airflow images that are pulled from _either_ Docker re
+
+For users, this change will result in a quicker, more efficient and more graceful deploy process.
+
+> **Note:** This improvemenet is only applicable to those running Airflow with the Celery Executor on Astronomer. For more information on Airflow Executors, refer to ["Airflow Executors Explained"](https://www.astronomer.io/guides/airflow-executors-explained) or [Configure an Airflow Deployment on Astronomer](https://www.astronomer.io/docs/cloud/stable/deploy/configure-deployment).
```
```suggestion
> **Note:** This improvement applies only to those running Airflow with the Celery Executor on Astronomer. For more information on Airflow Executors, refer to ["Airflow Executors Explained"](https://www.astronomer.io/guides/airflow-executors-explained) or [Configure an Airflow Deployment on Astronomer](https://www.astronomer.io/docs/cloud/stable/deploy/configure-deployment).
```

jwitz commented on cloud/stable/06_resources/01_release-notes.md:
```diff
@@ -138,7 +154,7 @@ Release Date: October 29, 2020
+- BugFix: Reset password flow triggers incorrect email for users who use local username/password to authenticate
 
 ### v0.21.1
 
```
Doesn't look like it will let me suggest changes to unedited sections of the doc, but there are two additional typos here. Ctrl +f "Workpace" and "Deploment" to find them.

paolaperaza replied:
Yikes good catch - applied change to Deployment typo but actually couldn't find Workspace typo. Will DM you to make sure I caught it!

jwitz commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,129 @@ description: "Astronomer Enterprise release notes."
+If you're looking to upgrade to Astronomer v0.23 from Astronomer v0.16, refer to [Upgrade to Astronomer v0.23](/docs/enterprise/stable/manage-astronomer/upgrade-to-0-23/).
 
-We're committed to testing all quarterly Astronomer Enterprise versions for scale, reliability and security on EKS, GKE and AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](support.astronomer.io).
+For instructions on how to upgrade to a patch version within the Astronomer v0.23 series, refer to [Upgrade to a Patch Version of Astronomer Enterprise](/docs/enterprise/stable/manage-astronomer/upgrade-astronomer/). 
```
```suggestion
For instructions on how to upgrade to a patch version within the Astronomer v0.23 series, refer to [Upgrade to a Patch Version of Astronomer Enterprise](/docs/enterprise/stable/manage-astronomer/upgrade-patch/). 
```

jwitz commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,129 @@ description: "Astronomer Enterprise release notes."
+We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](support.astronomer.io).
 
-Latest Patch Release: **v0.16.12**
+> **Note:** The perceived version gap between Astronomer Enterprise v0.16 and v0.23 is due to the nature of Astronomer's release schedule. To optimize for security and reliability, Astronomer Cloud releases are made available to Enterprise users only after they've passed a dedicated testing process and have been running successfully for a pre-determined amount of time. Astronomer Enterprise v0.23 ...
```
```suggestion
> **Note:** The perceived version gap between Astronomer Enterprise v0.16 and v0.23 is due to the nature of Astronomer's release schedule. To optimize for security and reliability, Astronomer Cloud releases are made available to Enterprise users only after they've passed a dedicated testing process. Astronomer Enterprise v0.23 includes _all_ changes made available on Astronomer Cloud between v0.16 and v0.23, in addition to Enterprise-only functionality.
```
To me, the runtime is part of the testing - if we don't provide more info on it here, I think it's best to cut - don't mind either way tho.

jwitz commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,129 @@ description: "Astronomer Enterprise release notes."
 
-- BugFix: Fix conditionally disabling subcharts
+- Support for Multiple Schedulers
+- A mechanism to enforce that users migrate to Airflow 1.10.14 prior to upgrading to 2.0
```
```suggestion
- A mechanism to ensure that users migrate to Airflow 1.10.14 prior to upgrading to 2.0
```

paolaperaza commented on cloud/stable/06_resources/01_release-notes.md:
```diff
@@ -60,7 +66,7 @@ As of Astronomer v0.23.5, we:
 - Support Airflow's ["upgrade check"](https://airflow.apache.org/docs/apache-airflow/stable/upgrade-check.html) in the Astronomer CLI (`$ astro dev upgrade-check`)
 
-The migration path to Airflow 2.0 on Astronomer follows all recommendations published by the Apache Airflow project. We expect to publish an official "Upgrade to Airflow 2.0 on Astronomer" guide as soon as Airflow 2.0 is generally available.
+To get started, read [Upgrade to Apache Airflow 2.0 on Astronomer](https://www.astronomer.io/docs/cloud/stable/customize-airflow/upgrade-to-airflow-2). The migration path to Airflow 2.0 on Astronomer follows all recommendations published by the Apache Airflow project.
```
```suggestion
To get started, read [Upgrade to Apache Airflow 2.0 on Astronomer](https://www.astronomer.io/docs/cloud/stable/customize-airflow/upgrade-to-airflow-2). The migration path to Airflow 2.0 on Astronomer follows [recommendations published by the Apache Airflow project](https://airflow.apache.org/docs/apache-airflow/stable/upgrading-to-2.html).
```

jwitz commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,129 @@ description: "Astronomer Enterprise release notes."
 
-- Improvement: New Houston API mutation that allows users to enable or disable KEDA (`updateDeploymentKedaConfig`)
-- BugFix: Alerts from Astronomer not sent when an Airflow component breaches the resource threshold level defined in Prometheus
+To see the version of Airflow your Deployment is running, refer to the corresponding **Settings** page of the Astronomer UI. If you're interested in upgrading versions, you now initialize the upgrade process via the Astronomer UI or CLI for a guided experience. From there, you'll receive instructions to complete the process by updating the Astronomer Certified (AC) image in your `Dockerfile`.
```
```suggestion
To see the version of Airflow your Deployment is running, refer to its **Settings** page in the Astronomer UI. If you're interested in upgrading versions, you can now initialize the upgrade process via the Astronomer UI or CLI for a guided experience. From there, you'll receive instructions to complete the process by updating the Astronomer Certified (AC) image in your `Dockerfile`.
```

jwitz commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,129 @@ description: "Astronomer Enterprise release notes."
+Astronomer Enterprise v0.23 introduces Deployment-level permissions, a much-awaited feature for large teams running multiple Airflow Deployments on Astronomer.
 
-> **Note:** AC 1.10.12 will be the _last_ version to support an Alpine-based image. In an effort to standardize our offering and optimize for reliability, we'll exclusively build, test and support Debian-based images starting with AC 1.10.13. A guide for how to migrate from Alpine to Debian coming soon.
+Users can now configure and be assigned 1 of 3 user roles within each individual Airflow Deployment - _Admin_, _Editor_ and _Viewer_. If you operate with production and development environments, for example, you can restrict a user's access to your production Airflow Deployment as a _Viewer_ but grant them full access to your development Airflow Deployment as an _Admin_ all within the same Worksp ...
```
```suggestion
Within each individual Airflow Deployment, users can now configure and be assigned 1 of 3 user roles:  _Admin_, _Editor_, or _Viewer_. If you operate with production and development environments, for example, you can restrict a user's access to your production Airflow Deployment as a _Viewer_ but grant them full access to your development Airflow Deployment as an _Admin_ all within the same Workspace. Users who exclusively need Workspace permissions do not require permissions to any or all Airflow Deployments within it.
```

jwitz commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,129 @@ description: "Astronomer Enterprise release notes."
+While this change largely sets the foundation for new features in later releases, Astronomer v0.23 includes:
 
-#### Bug Fixes and Improvements
+- A new banner in the **Deployment** view of the Astronomer UI if a deploy to that Airflow Deployment is in progress.
```
```suggestion
- A new banner in the **Deployment** view of the Astronomer UI that indicates if a deploy is in progress.
```

jwitz commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,129 @@ description: "Astronomer Enterprise release notes."
-#### Bug Fixes and Improvements
+- A new banner in the **Deployment** view of the Astronomer UI if a deploy to that Airflow Deployment is in progress.
+- Refined logic for **Deployment Health Status** (Unhealthy/Red, Healthy/Green, Deploying/Blue and Unknown/Gray) that's visible as a "bubble" next to all Airflow Deployments in the Astronomer UI.
+- A set of error messages to alert you if a deploy has failed or was otherwise _not_ complete.
```
```suggestion
- A set of error messages to alert you if a deploy failed or was otherwise not completed.
```

jwitz commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,129 @@ description: "Astronomer Enterprise release notes."
+Previously, the number of Celery Worker replicas that could be immediately created following a deploy was restricted to 25% of the total number of desired Celery Workers, until the original Workers shut down. This meant that it took longer for replacement Celery Workers to be created and start picking up Airflow tasks. Defined as `maxSurge`, that percentage is now set to 100%, allowing the maximu ...
 
-Release Date: September 25, 2020
+For users, this change will result in a quicker, more efficient and more graceful deploy process.
```
```suggestion
For users, this change will result in a quicker, more efficient, and more graceful deploy process.
```

jwitz commented on enterprise/next/09_resources/01_release-notes.md:
```diff
@@ -6,287 +6,129 @@ description: "Astronomer Enterprise release notes."
+For users, this change will result in a quicker, more efficient and more graceful deploy process.
 
-#### Export Logs from Fluentd to Amazon S3
+> **Note:** This improvemenet is only applicable to those running Airflow with the Celery Executor on Astronomer. For more information on Airflow Executors, refer to ["Airflow Executors Explained"](https://www.astronomer.io/guides/airflow-executors-explained) or [Configure an Airflow Deployment on Astronomer](https://www.astronomer.io/docs/enterprise/stable/deploy/configure-deployment).
```
```suggestion
> **Note:** This improvement applies only to those running Airflow with the Celery Executor on Astronomer. For more information on Airflow Executors, refer to ["Airflow Executors Explained"](https://www.astronomer.io/guides/airflow-executors-explained) or [Configure an Airflow Deployment on Astronomer](https://www.astronomer.io/docs/enterprise/stable/deploy/configure-deployment).
```

jwitz reviewed (commented):
Some minor fixes and comments, but otherwise looks good @paolaperaza !

paolaperaza commented:
Thanks for the review @jwitz , @vishwas-astro , @astrorfox ! Merging 🎉
